asv/commands/run.py

```diff
@@ -499,10 +506,18 @@ def iter_rounds_commits():
 
                         if not skip_save:
                             result.save(conf.results_dir)
+                        if strict:
+                            failed = [
+                                failed_bench for failed_bench in
+                                result.errcode if
+                                result.errcode[failed_bench] != 0]
+                            failures += failed
 
                         if durations > 0:
                             duration_set = Show._get_durations([(machine, result)], benchmark_set)
                             log.info(cls.format_durations(duration_set[(machine, env.name)], durations))
+        if failures and strict:
+            return len(failures)
```

Comment:
That'll exit with success if you have 256*n failures (sys.exit takes lowest bits).

Probably should be: `2 if failures else 0` (maybe different error code than default is preferable).

Reply:
Fixed in: https://github.com/airspeed-velocity/asv/pull/865/commits/324ba8016bd9a4369a40d35a2b19a703852c9ab2